services/press-fault-data-simulator-service/app/main.py

```
@@ -0,0 +1,33 @@
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+
+from app.routers import simulator_router
+from app.utils.logger import system_log
+from app.routers.simulator_router import scheduler_service
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+
+    system_log.info(f"애플리케이션 시작 준비 완료.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)